src/level_four/tap_cards/controller/controller_tap_cards.h

```diff
@@ -29,20 +30,22 @@
  * @note
  */
 typedef struct NFC_connection_data {
-  uint8_t desktop_control;
-  uint8_t active_cmd_type;
+  uint8_t desktop_control;    // TODO: Remove deprecated
+  uint8_t active_cmd_type;    // TODO: Remove deprecated
   int8_t keystore_index;
   uint8_t acceptable_cards;
   uint8_t tapped_card;
-  uint8_t lvl3_retry_point;
-  uint8_t lvl4_retry_point;
-  uint8_t resume_point;
+  uint8_t lvl3_retry_point;    // TODO: Remove deprecated
+  uint8_t lvl4_retry_point;    // TODO: Remove deprecated
+  uint8_t resume_point;        // TODO: Remove deprecated
   uint8_t retries;
   uint8_t family_id[FAMILY_ID_SIZE +
                     2];    // TODO: Review(need to find reason for extra byte)
   uint8_t card_key_id[4];
   uint8_t recovery_mode;
   uint8_t card_absent_retries;
+  bool pairing_required;    /// Input config set before card initialization to
```

Comment:
a more appropriate name would be `init_secret_channel`

Reply:
Please check the last [commit](https://github.com/Cypherock/x1_wallet_firmware/pull/272/commits/0659e181f80a98561c051325bf3109e2f67f876e) where this change has been removed.

The reasoning for the change:
- It's not a task of applet init API to load pairing keys.
- Any function requiring an encrypted channel can call a global function to load keys and/or check pairing.